src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/styles.cljs

```diff
@@ -1,15 +1,30 @@
 (ns status-im.ui.screens.offline-messaging-settings.edit-mailserver.styles
-  (:require-macros [status-im.utils.styles :refer [defstyle]]))
+  (:require-macros [status-im.utils.styles :refer [defstyle]])
+  (:require [status-im.ui.components.styles :as styles]))
 
-(def edit-mailserver-view
+(defstyle edit-mailserver-view
   {:flex              1
    :margin-horizontal 16
    :margin-vertical   15})
 
-(def input-container
-  {:margin-bottom 15})
+(defstyle input-container
+  {:flex-direction    :row
+   :align-items       :center
+   :justify-content   :space-between
+   :border-radius     styles/border-radius
+   :height            52
+   :margin-top        15})
+
+(defstyle input
+  {:flex               1
+   :font-size          15
+   :letter-spacing     -0.2
+   :android            {:padding 0}})
+
+(defstyle qr-code
+  {:margin-right 14})
 
-(def bottom-container
+(defstyle bottom-container
```

Comment:
Any reason for this?

Reply:
I was under the impression we were using `defstyle` by default instead of `def`, I will change them back